whisper/whisperv6/whisper.go

```diff
@@ -127,10 +130,37 @@ func New(cfg *Config) *Whisper {
 }
 
 func (w *Whisper) MinPow() float64 {
-	val, _ := w.settings.Load(minPowIdx)
+	val, exist := w.settings.Load(minPowIdx)
+	if !exist || val == nil {
+		return DefaultMinimumPoW
+	}
+	return val.(float64)
```

Comment:
If unchecked, type assertions will panic. I would prefer something like:

```
v, ok := val.(float64)
if !ok {
  fmt.Println("Error, using default")
  return DefaultMinimumPoW
}
return v
```

or change the function's signature to be `(float64, error)`, which is better imho

Reply:
changed as you suggested. changing the signature would be an overkill: Store/Load functions either work or don't, no need for runtime check. if they don't, the tests will show it.